backend_new/test/unit/services/translation.service.spec.ts

```diff
@@ -0,0 +1,269 @@
+      updatedAt: new Date(),
+      name: 'jurisdiction',
+      languages: [],
+      multiselectQuestions: [],
+      publicUrl: '',
+      emailFromAddress: '',
+      rentalAssistanceDefault: '',
+      enableAccessibilityFeatures: true,
+      enableUtilitiesIncluded: true,
+    },
+    units: [],
+    unitsSummarized: undefined,
+    unitsSummary: [],
+    showWaitlist: true,
+    referralApplication: undefined,
+  };
+  return {
+    ...basicListing,
+    unitAmenities: 'untranslated unit amenities',
+    smokingPolicy: 'untranslated smoking policy',
+    servicesOffered: 'untranslated services offered',
+    petPolicy: 'untranslated pet policy',
+    neighborhood: 'untranslated neighborhood',
+    amenities: 'untranslated amenities',
+    accessibility: 'untranslated accessibility',
+    whatToExpect: 'untranslated what to expect',
+    specialNotes: 'untranslated special notes',
+    requiredDocuments: 'untranslated required documents',
+    rentalHistory: 'untranslated rental history',
+    rentalAssistance: 'untranslated rental assistance',
+    programRules: 'untranslated program rules',
+    criminalBackground: 'untranslated criminal background',
+    creditHistory: 'untranslated credit history',
+    costsNotIncluded: 'untranslated costs not included',
+    applicationPickUpAddressOfficeHours:
+      'untranslated application pick up address office hours',
+    listingMultiselectQuestions: [
+      {
+        multiselectQuestions: {
+          id: 'multiselectQuestions id 1',
+          createdAt: new Date(),
+          updatedAt: new Date(),
+          text: 'untranslated multiselect text',
+          description: 'untranslated multiselect description',
+          subText: 'untranslated multiselect subtext',
+          optOutText: 'untranslated multiselect opt out text',
+          listings: [],
+          jurisdictions: [],
+          applicationSection:
+            MultiselectQuestionsApplicationSectionEnum.preferences,
+        },
+        listings: basicListing,
+      },
+    ],
+  };
+};
+
+const translatedStrings = [
+  'translated application pick up address office hours',
+  'translated costs not included',
+  'translated credit history',
+  'translated criminal background',
+  'translated program rules',
+  'translated rental assistance',
+  'translated rental history',
+  'translated required documents',
+  'translated special notes',
+  'translated what to expect',
+  'translated accessibility',
+  'translated amenities',
+  'translated neighborhood',
+  'translated pet policy',
+  'translated services offered',
+  'translated smoking policy',
+  'translated unit amenities',
+  'translated multiselect text',
+  'translated multiselect description',
+  'translated multiselect subtext',
+  'translated multiselect opt out text',
+];
+
+describe('Testing translations service', () => {
+  let service: TranslationService;
+  let prisma: PrismaService;
+  let googleTranslateServiceMock;
+  beforeEach(async () => {
```

Comment:
can we switch this to `beforeAll` instead

Reply:
I am unable to switch to beforeAll because I change the mocks in each test and the `toHaveBeenCalledTimes` is inaccurate unless it is reset every test